src/trio/socket.py

```diff
@@ -70,10 +70,17 @@
 if sys.implementation.name == "cpython":
     from socket import (
         if_indextoname as if_indextoname,
-        if_nameindex as if_nameindex,
         if_nametoindex as if_nametoindex,
     )
 
+    # In https://github.com/kivy/python-for-android, if_nameindex
```

Comment:
For what it's worth: as far as I can tell, this is just a CPython thing. Someone filed an (marked as spam?) bug report on CPython related to it https://bugs.python.org/msg279495

Reply:
I think 9dda073 should be sufficient